src/GraphingImpl/Mocks/GraphRenderer.h

```diff
@@ -98,6 +98,11 @@ namespace MockGraphingImpl
             return S_OK;
         }
 
+        virtual HRESULT PrepareGraph()
```

Comment:
Hovering over lines on the graph and variable editing are broken in this PR/new engine.

Reply:
I can't repro these issues.

Reply:
I was able to repro this for y=mx, I've reached out to the graphing team.